<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>601</width>
      <height>391</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="layout">
     <item>
      <widget class="QScrollArea" name="scrollArea">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>597</width>
          <height>387</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionImport"/>
    <addaction name="actionSmart_Import"/>
    <addaction name="actionResize"/>
    <addaction name="actionSave"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <widget class="QMenu" name="menuTransform">
     <property name="title">
      <string>Transform</string>
     </property>
     <addaction name="actionRotate_Transpose"/>
     <addaction name="actionRotate_Clockwise"/>
     <addaction name="actionRotate_Counterclockwise"/>
     <addaction name="actionRotate_180"/>
     <addaction name="actionFlipX"/>
     <addaction name="actionFlip_Y"/>
    </widget>
    <addaction name="actionRepaint_All"/>
    <addaction name="menuTransform"/>
   </widget>
   <widget class="QMenu" name="menuZoom">
    <property name="title">
     <string>Zoom</string>
    </property>
    <addaction name="actionZoom_In"/>
    <addaction name="actionZoom_Out"/>
    <addaction name="actionReset_Zoom"/>
   </widget>
   <widget class="QMenu" name="menuColor">
    <property name="title">
     <string>Color</string>
    </property>
    <addaction name="actionBrush_Color"/>
    <addaction name="actionEraser"/>
    <addaction name="actionColor_Count"/>
    <addaction name="actionBackground_Color"/>
    <addaction name="actionOutline_Color"/>
    <addaction name="actionColor_Pallette"/>
   </widget>
   <widget class="QMenu" name="menuPatterns">
    <property name="title">
     <string>Patterns</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuZoom"/>
   <addaction name="menuColor"/>
   <addaction name="menuPatterns"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
   <property name="toolTip">
    <string>Create a new image grid</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open an existing image</string>
   </property>
  </action>
  <action name="actionImport">
   <property name="text">
    <string>Import</string>
   </property>
   <property name="toolTip">
    <string>Open an existing image and scale it</string>
   </property>
  </action>
  <action name="actionSmart_Import">
   <property name="text">
    <string>Smart Import</string>
   </property>
   <property name="toolTip">
    <string>Open an existing image and scale it smartly using the opened pattern</string>
   </property>
  </action>
  <action name="actionResize">
   <property name="text">
    <string>Resize Image</string>
   </property>
   <property name="toolTip">
    <string>Resize current image</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save the opened image</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
   <property name="toolTip">
    <string>Quit the program</string>
   </property>
  </action>
  <action name="actionRepaint_All">
   <property name="text">
    <string>Repaint Image</string>
   </property>
   <property name="toolTip">
    <string>Repaints the entire image</string>
   </property>
  </action>
  <action name="actionRotate_Transpose">
   <property name="text">
    <string>Transpose Image</string>
   </property>
   <property name="toolTip">
    <string>Transposes the source image file</string>
   </property>
  </action>
  <action name="actionRotate_Clockwise">
   <property name="text">
    <string>Rotate Clockwise</string>
   </property>
   <property name="toolTip">
    <string>Rotates the source image file by 90 degrees clockwise</string>
   </property>
  </action>
  <action name="actionRotate_Counterclockwise">
   <property name="text">
    <string>Rotate Counter-Clockwise</string>
   </property>
   <property name="toolTip">
    <string>Rotates the source image file by 90 degrees counter-clockwise</string>
   </property>
  </action>
  <action name="actionRotate_180">
   <property name="text">
    <string>Rotate 180</string>
   </property>
   <property name="toolTip">
    <string>Rotates the source image file by 180 degrees</string>
   </property>
  </action>
  <action name="actionFlipX">
   <property name="text">
    <string>Reflect X axis</string>
   </property>
   <property name="toolTip">
    <string>Reflects the source image files X axis</string>
   </property>
  </action>
  <action name="actionFlip_Y">
   <property name="text">
    <string>Reflect Y axis</string>
   </property>
   <property name="toolTip">
    <string>Reflects the source image files Y axis</string>
   </property>
  </action>
  <action name="actionZoom_In">
   <property name="text">
    <string>Zoom In</string>
   </property>
   <property name="toolTip">
    <string>Increases size of image by 10%</string>
   </property>
  </action>
  <action name="actionZoom_Out">
   <property name="text">
    <string>Zoom Out</string>
   </property>
   <property name="toolTip">
    <string>Decreases size of image by 10%</string>
   </property>
  </action>
  <action name="actionReset_Zoom">
   <property name="text">
    <string>Reset Zoom</string>
   </property>
   <property name="toolTip">
    <string>Resets the zoom to 100%</string>
   </property>
  </action>
  <action name="actionBrush_Color">
   <property name="text">
    <string>Brush Color</string>
   </property>
   <property name="toolTip">
    <string>Select a new brush color</string>
   </property>
  </action>
  <action name="actionEraser">
   <property name="text">
    <string>Eraser</string>
   </property>
   <property name="toolTip">
    <string>Erase clicked tiles</string>
   </property>
  </action>
  <action name="actionColor_Count">
   <property name="text">
    <string>Count Colors</string>
   </property>
   <property name="toolTip">
    <string>Count colors in the image</string>
   </property>
  </action>
  <action name="actionBackground_Color">
   <property name="text">
    <string>Background</string>
   </property>
   <property name="toolTip">
    <string>Select a new background color</string>
   </property>
  </action>
  <action name="actionOutline_Color">
   <property name="text">
    <string>Outline</string>
   </property>
   <property name="toolTip">
    <string>Select a new outline color</string>
   </property>
  </action>
  <action name="actionColor_Pallette">
   <property name="text">
    <string>Palette Change</string>
   </property>
   <property name="toolTip">
    <string>Limit colors to the current image</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
   <property name="toolTip">
    <string>Description of this program</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
